src/frontend/src/lib/components/signer/SignerAlert.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import IconCheck from '$lib/components/icons/IconCheck.svelte';
+	import IconClose from '$lib/components/icons/IconClose.svelte';
+
+	export let type: 'ok' | 'error';
```

Comment:
not a call to action: maybe something less built-in as `type`? `alertType`, or similar

Reply:
Good call. `alertType` sounds good.

Reply:
Done.